lib/environment_light.rb

```diff
@@ -11,5 +11,8 @@
 require 'bundler/setup'
 
 # Load application settings
-require File.expand_path("../../config/initializers/rails_config", __FILE__)
-Settings = RailsConfig.load_files ["settings.yml","settings.local.yml", "settings/#{Rails.env}.yml"].map{|f| "#{Rails.root}/config/#{f}" }
+require Rails.root.join('config/initializers/rails_config.rb')
+# only load basic files, NOT the auxiliaries like hets.yml
+settings_files = RailsConfig.setting_files(Rails.root.join('config'), Rails.env)
+abs_settings_files = settings_files.map { |f| Rails.root.join('config', f) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.